api/__init__.py

```diff
@@ -0,0 +1,49 @@
+from typing import Dict, Iterator, Type, Tuple, List
+import pathlib
+import importlib
+import sys
+import logging
+
+import pydantic
+
+_logger = logging.getLogger(__name__)
+
+SCHEMAS_PATH = pathlib.Path(__file__).parent / "schemas"
+
+
+def load_public_schemas() -> List[Type[pydantic.BaseModel]]:
+    """Loads all pydantic Schemas for export.
```

Comment:
"Loads" => "Finds"
"Schemas" => "model classes (i.e. that derive from pydantic.BasModel)" ?
Drop "for export" since that's an assumption about how this function will be used.